explorer/mempool.go

```diff
@@ -386,3 +387,41 @@ func (votes byHeight) Len() int {
 func (votes byHeight) Swap(i, j int) {
 	votes[i], votes[j] = votes[j], votes[i]
 }
+
+// matchMempoolVins filters relevant mempool transaction inputs
```

Comment:
For consistency, we always wrap after 80 columns.  I just use a wrapping extension (Rewrap for VS Code).

Reply:
For any Atom users that might read this, the equivalent package is `autoflow`, installed by default, and uses `shift+ctrl+q` to wrap the selection at 80 lines (configurable).